int16 taskId
int16 taskState
